### PR TITLE
Adding access rights UI for service bus bindings.

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
#160

File: Bindings/bindings.json [hunk 1]
```diff
@@ -475,6 +475,22 @@
                     "required": true,
                     "label": "Service Bus connection",
                     "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights."
+                },
```
Comment (mathewc) [inline Bindings/bindings.json]: Why does the above show resource=EventHub?

File: Bindings/bindings.json [hunk 2]
```diff
@@ -475,6 +475,22 @@
...
+                            "value": "Manage",
+                            "display": "Manage"
+                        },
+                        {
+                            "value": "Listen",
+                            "display": "Listen"
+                        }
+                    ],
+                    "label": "AccessRights",
+                    "help": "This is the access right the client has to the queue or topic subscription."
```
Comment (mathewc) [inline Bindings/bindings.json]: nit: "access rights"

File: Bindings/bindings.json (hunk 1 above)
Comment (soninaren) [inline Bindings/bindings.json]: Have an issue filed in portal for it. 
https://github.com/projectkudu/AzureFunctionsPortal/issues/329

resource=EventHub actually opens service bus connection blade. It needs to be renamed.

**Comment by mathewc** [comment]
:ship:

**Pull request merged**